tokenizers/src/processors/template.rs

```diff
@@ -647,6 +648,26 @@ impl PostProcessor for TemplateProcessing {
             add_special_tokens,
         )
     }
+
+    fn process_chain(
+        &self,
+        encodings: Vec<Encoding>,
+        add_special_tokens: bool,
+    ) -> Result<Vec<Encoding>> {
+        let is_pair = encodings.len() == 2;
+        if is_pair {
```

Comment:
wdyt about this implementation & hardcoded number 2. 
Basically, I'm saying: if there are 2 elements in `encodings` vector, then run `self.process(encoding, pair_encoding)` with **pair_encoding** encoding.

otherwise, run `self.process(encoding, pair_encoding=None)` for each element in `encodings`